### PR TITLE
[fetch] Specify unique names for tests

### PR DESCRIPTION
**Pull request by jugglinmike**
Duplicated test names make interpreting results difficult for humans and machines. In order to avoid this confusion, [an upcoming extension to the test harness](https://github.com/w3c/testharness.js/pull/240) will cause any test file with duplicate test names to be interpreted as an error.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Chrome (unstable channel) #
Testing web-platform-tests at revision 8236114a18545584c8f982790073e1944403efad

**Comment by w3c-bots** [comment]
# Firefox (nightly channel) #
Testing web-platform-tests at revision 8236114a18545584c8f982790073e1944403efad

File: fetch/api/request/request-cache-default-conditional.html [hunk 1]
```diff
@@ -14,15 +14,15 @@
     <script>
     var tests = [
       {
-        name: 'RequestCache "default" mode with an If-Modified-Since header is treated similarly to "no-store"',
+        name: 'RequestCache "default" mode with an If-Modified-Since header (following a request without headers) is treated similarly to "no-store"',
```
Comment (jgraham) [inline fetch/api/request/request-cache-default-conditional.html]: Presumably the request does have some headers, just not extra ones.

Comment (jugglinmike) [inline fetch/api/request/request-cache-default-conditional.html]: Fair enough. I've changed the text from "without headers" to "without additional headers"

**Pull request merged**